### ISSUE TITLE
Could not load type 'System.Management.Automation.Subsystem.PredictionResult' on PS 7.2.0

### ISSUE THREAD

**Issue by fgilde** [PowerShell/PSReadLine#2966]
### Prerequisites

- [X] Write a descriptive title.
- [X] Make sure you are able to repro it on the [latest released version](https://www.powershellgallery.com/packages/PSReadLine)
- [X] Search the existing issues, especially the pinned issues.

### Exception report

```console
Powershell wants that I report the Bug here, so I will do. I have updated to PowerShell 7.2.0 and now the `PSReadLine: 2.2.0-beta1` is not only no more working it also crashes PowerShell  

Complete Output

Oops, something went wrong.
Please report this bug with ALL the details below, including both the 'Environment' and 'Exception' sections.
Please report on GitHub: https://github.com/PowerShell/PSReadLine/issues/new?template=Bug_Report.md
Thank you!

### Environment
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.22000
BufferWidth: 120
BufferHeight: 30

Last 0 Keys


```

### Exception
```
System.TypeLoadException: Could not load type 'System.Management.Automation.Subsystem.PredictionResult' from assembly 'Microsoft.PowerShell.PSReadLine.Polyfiller, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null'.
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionViewBase.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.PredictionInlineView.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Prediction.Reset()
   at Microsoft.PowerShell.PSConsoleReadLine.Initialize(Runspace runspace, EngineIntrinsics engineIntrinsics)
   at Microsoft.PowerShell.PSConsoleReadLine.ReadLine(Runspace runspace, EngineIntrinsics engineIntrinsics, CancellationToken cancellationToken)
```

```
```

### Screenshot

![image](https://user-images.githubusercontent.com/11070717/140900364-f053fde5-c836-44b6-b725-109adce1d83c.png)

![image](https://user-images.githubusercontent.com/11070717/140900885-dd471c0f-6ad4-442b-b402-906994008c10.png)

### Environment data

```console
PSReadLine: 2.2.0-beta1
PowerShell: 7.2.0
OS: Microsoft Windows 10.0.22000
BufferWidth: 227
BufferHeight: 49
```

### Steps to reproduce

Not sure but have a powershell 7.1.5 installed with the PSReadLine: 2.2.0-beta1 Module
Update to powershell 7.2.0
restart powershell

### Expected behavior

No Exception

### Actual behavior

Exception

**Comment by alberthoekstra** [comment]
Installing 2.2.0-beta3 fixed that issue for me.

```
 Install-Module -Name PSReadLine -RequiredVersion 2.2.0-beta3 -AllowPrerelease
```

**Comment by iSazonov** [comment]
Latest is Beta4

**Comment by daxian-dbw** [comment]
This issue was fixed in 2.2.0-beta3 version of PSReadLine. You can fix this by upgrading to the latest [2.2.0-beta4 version of PSReadLine](https://www.powershellgallery.com/packages/PSReadLine/2.2.0-beta4). Instructions for doing so:
1. stop all instances of `pwsh`.
2. from `cmd.exe` run: `pwsh -noprofile -command "Install-Module PSReadLine -AllowPrerelease -Force"`

--------

If you want to remove that beta version of PSReadLine and use the 2.1.0 version of PSReadLine that's shipped with PowerShell 7.2, you can:
1. run `pwsh -noprofile -noninteractive` to start `pwsh` without loading PSReadLine
2. run `Uninstall-Module -Name PSReadLine -RequiredVersion <2.2.0-beta1 or 2.2.0-beta2> -AllowPrerelease` to remove the module. Or, you can manually remove that module folder.
